adbe/adb_enhanced.py

```diff
@@ -1496,7 +1496,7 @@ def perform_install(file_path):
 @ensure_package_exists
 def perform_uninstall(app_name):
     print_verbose('Uninstalling %s' % app_name)
-    return_code, _, stderr = execute_adb_command2('uninstall %s' % app_name)
+    return_code, _, stderr = execute_adb_shell_command2('pm uninstall --user 0 %s' % app_name)
```

Comment:
The two commands are not the same - https://www.xda-developers.com/uninstall-carrier-oem-bloatware-without-root-access/

I think this should be an optional sub-command

Reply:
It's much better

Reply:
I merged the PR but I wonder if we should do "-0" as an automatic fallback?

Reply:
Do it need a user confirmation for the second command ?

Reply:
No. Any user confirmations will make it unscriptable. I have avoided that in this tool completely.

Reply:
Should I create a new PR or can we reopen this one ?

Reply:
This is already merged, so, a new one.